gto/git_utils.py

```diff
@@ -32,16 +32,21 @@ def wrapped_f(*args, **kwargs):
 
         return f(**kwargs)
 
-    def _turn_args_into_kwargs(
-        args: tuple, kwargs: Dict[str, object]
-    ) -> Dict[str, object]:
-        kwargs_complement = {
-            k: args[i]
-            for i, k in enumerate(inspect.getfullargspec(f).args)
-            if i < len(args)
-        }
-        kwargs.update(kwargs_complement)
-        return kwargs
+    return wrapped_f
+
+
+def auto_push_on_remote_repo(f: Callable):
```

Comment:
@aguschin I was thinking to rename this as `git_clone_and_auto_push_on_remote_repo`. What do you think?

Reply:
I think `auto_push` is good since you have an argument that you're setting to `True` with this wrapper.

Reply:
ok, you convinced me :)